tests/test_scene_rendering/test_scene.py

```diff
@@ -0,0 +1,49 @@
+import os
+import pytest
+import subprocess
+from manim import file_writer_config
+
+from ..utils.commands import capture
+from ..utils.video_tester import *
+
+
+@pytest.mark.slow
+@video_comparison(
+    "SquareToCircleWithDefaultValues.json",
+    "videos/simple_scenes/1440p60/SquareToCircle.mp4",
+)
+def test_basic_scene_with_default_values(tmp_path, manim_cfg_file, simple_scenes_path):
+    scene_name = "SquareToCircle"
+    command = [
+        "python",
+        "-m",
+        "manim",
+        simple_scenes_path,
+        scene_name,
+        "--media_dir",
+        str(tmp_path),
+    ]
+    out, err, exit_code = capture(command)
+    assert exit_code == 0, err
+    # print(out)
+    # print(out)
+    # assert os.path.isfile()
+
+
+@video_comparison(
+    "SquareToCircleWithPFlag.json", "videos/simple_scenes/480p15/SquareToCircle.mp4"
+)
+def test_basic_scene_p_flag(tmp_path, manim_cfg_file, simple_scenes_path):
+    scene_name = "SquareToCircle"
+    command = [
+        "python",
+        "-m",
+        "manim",
+        simple_scenes_path,
+        scene_name,
+        "-l",
+        "--media_dir",
+        str(tmp_path),
+    ]
```

Comment:
Issue: this doesn't seem to be using the -p flag.

Reply:
The `-p` flag won't work on headless machines like the Github Actions runners, since they don't have video players.

Reply:
Right, that makes sense. But I thought this test was supposed to be testing the -p flag. It's named `test_basic_scene_p_flag` after all.

Reply:
yes, it's my fault, it's a typo. By `-p` flag I meant `-l` flag.